packages/types/nft.ts

```diff
@@ -1,6 +1,6 @@
 import { ComponentType, ReactNode, RefAttributes } from 'react'
 
-import { GenericToken } from '.'
+import { Duration, GenericToken } from '.'
```

Comment:
can you change this import to not import from the index? ie import `GenericToken` from `./token` and `Duration` from its own file too